app/services/export/report.rb

```diff
@@ -55,6 +58,7 @@ def rows
       row << implementing_organisations_rows.fetch(activity.id, nil)
       row << partner_organisation_rows.fetch(activity.id, nil)
       row << change_state_rows.fetch(activity.id, nil)
+      row << commitment_rows.fetch(activity.id, nil)
```

Comment:
I think the way we've coded it now, there will always be a value (which could be the empty string), but if there wasn't, do we want it to return nil, or the empty string?

(I'm not sure, that's why I'm asking!)

Reply:
Yeah, I am not sure - I guess it depends on if when output to csv it handles the nil, in which case I prefer the nil - I will have to look.

Reply:
It seems to be fine with nil, so going with that.